src/main/java/ch/njol/skript/effects/EffInvisible.java

```diff
@@ -0,0 +1,70 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ * Copyright Peter Güttinger, SkriptLang team and contributors
+ */
+package ch.njol.skript.effects;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.Effect;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.util.Kleenean;
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+@Name("Make Invisible")
+@Description({"Makes a living entity visible/invisible, achieving the same result as an invisibility potion but without a time limit.",
+	"When changing an entity's invisibility while using an invisibility potion on it, the potion will be overridden and after it runs out the entity will keep its invisibility."})
```

Comment:
Does this effect add potioh particles around the entity? if not it's better to mention that
I've also made some rephrasing to the description, double check of the information is still correct.
```suggestion
@Description({
	"Makes a living entity visible/invisible. This effect has the same result as an invisibility potion but without a time limit.",
	"When setting an entity to invisible while using an invisibility potion on it, the potion will be overridden and when runs out the entity keeps its invisibility."
})
```

Reply:
I went ahead and rephrased the second sentence to make sure it is understandable that this is *not* a potion.